contracts/lupi.sol

```diff
@@ -3,9 +3,14 @@ pragma solidity ^0.4.11;
 import "./Owned.sol";
```

Comment:
I guess this pattern is used as a safeguard, eg. if you suspect your keys may be compromised, you can transfer to a fresh account (before the hackers do :)

Any references on the origins of this pattern? Let's understand why it's used, before cargo culting it in.